lib/ansible/modules/system/aix_lvol.py

```diff
@@ -0,0 +1,362 @@
+    if new_size < x:
+        new_size += base
+    return new_size
+
+
+def parse_lv(data):
+    name=None
+
+    for line in data.splitlines():
+        match = re.search("LOGICAL VOLUME:\s+(\w+)\s+VOLUME GROUP:\s+(\w+)", line)
+        if match is not None:
+            name = match.group(1)
+            vg = match.group(2)
+            continue
+        match = re.search("LPs:\s+(\d+).*PPs", line)
+        if match is not None:
+            lps = int(match.group(1))
+            continue
+        match = re.search("PP SIZE:\s+(\d+)", line)
+        if match is not None:
+            pp_size = int(match.group(1))
+            continue
+        match = re.search("INTER-POLICY:\s+(\w+)", line)
+        if match is not None:
+            policy = match.group(1)
+            continue
+
+    if not name:
+        return None
+
+    size = lps * pp_size
+
+    return {'name': name, 'vg': vg, 'size': size, 'policy': policy}
+
+def parse_vg(data):
+
+    for line in data.splitlines():
+
+        match = re.search("VOLUME GROUP:\s+(\w+)", line)
+        if match is not None:
+            name = match.group(1)
+            continue
+
+        match = re.search("TOTAL PP.*\((\d+)", line)
+        if match is not None:
+            size = int(match.group(1))
+            continue
+
+        match = re.search("PP SIZE:\s+(\d+)", line)
+        if match is not None:
+            pp_size = int(match.group(1))
+            continue
+
+        match = re.search("FREE PP.*\((\d+)", line)
+        if match is not None:
+            free = int(match.group(1))
+            continue
+
+    return {'name': name, 'size': size, 'free': free, 'pp_size': pp_size}
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            vg=dict(required=True),
+            lv=dict(required=True),
+            lv_type=dict(required=False),
+            size=dict(type='str'),
+            opts=dict(type='str'),
+            copies=dict(type='str'),
+            state=dict(choices=["absent", "present"], default='present'),
+            policy=dict(choices=["maximum", "minimum"], default='maximum'),
+            pvs=dict(type='str')
+        ),
+        supports_check_mode=True,
+    )
+
+    vg = module.params['vg']
+    lv = module.params['lv']
+    lv_type = module.params['lv_type']
+    size = module.params['size']
+    opts = module.params['opts']
+    copies = module.params['copies']
+    policy = module.params['policy']
+    state = module.params['state']
+    pvs = module.params['pvs']
+
+    if pvs is None:
+        pvs = ""
+    else:
+        pvs = pvs.replace(",", " ")
+
+    if opts is None:
+        opts = ""
+
+    if copies is None:
+        copies = "1"
+
+    if lv_type is None:
+        lv_type = "jfs2"
+
+    if policy == "maximum":
+        lv_policy="x"
+    else:
+        lv_policy="m"
+
+    # Add --test option when running in check-mode
+    if module.check_mode:
+        test_opt = 'echo '
```

Comment:
This doesn't seem to add `--test`, is the above comment out of date, or is this option name wrong?